minishift/kedge/auth-local.yml

```diff
@@ -1,16 +1,99 @@
 name: auth
+controller: DeploymentConfig
+
+revisionHistoryLimit: 2
+strategy:
+  rollingParams:
+    timeoutSeconds: 7200
+  type: Rolling
+triggers:
+- type: ConfigChange
+
+init-containers:
+- name: wait-for-auth-db
+  image: fabric8/fabric8-dependency-wait-service:v6632df1
+  imagePullPolicy: IfNotPresent
+  command:
+  - sh
+  - "-c"
+  - fabric8-dependency-wait-service-linux-amd64 postgres://auth@auth-db:5432
+  env:
+  - name: DEPENDENCY_POLL_INTERVAL
+    value: '1'
+  - name: DEPENDENCY_LOG_VERBOSE
+    value: 'true'
+
 containers:
-- image: fabric8/fabric8-auth:dev
+- name: auth
+  image: fabric8/fabric8-auth:[[ IMAGE_TAG ]]
+  imagePullPolicy: IfNotPresent
   env:
   - name: AUTH_DEVELOPER_MODE_ENABLED
-    value: "true"
-  - name: AUTH_POSTGRES_HOST
-    value: "db-auth"
-  - name: AUTH_POSTGRES_PORT
-    value: "5432"
+    value: [[ AUTH_DEVELOPER_MODE_ENABLED ]]
+  envFrom:
+  - configMapRef:
+      name: auth
+  - secretKeyRef:
+      name: auth
+
+  health:
```

Comment:
since liveness and readiness checks are exactly the same, so we can use the kedge's shortcut called health which populates the liveness and readiness both for us.